protocol/messenger_communities.go

```diff
@@ -470,7 +470,9 @@ func (m *Messenger) startCuratedCommunitiesUpdateLoop() {
 
 	const errorTimeout = 10 * time.Second
 	const successTimeout = 120 * time.Second
-	const unknownCommunitiesFoundTimeout = 3 * time.Second
+	// TODO lower this back again once the real curated community contract is up
+	// The current contract contains communities that are no longer accessible on waku
+	const unknownCommunitiesFoundTimeout = 60 * time.Second
 
 	go func() {
 		for {
```

Comment:
I can't comment below enough, but maybe we could make the `SendCuratedCommunitiesUpdate` function smarter too. Right now, it sends all the communities every time. Maybe it should only be called when there is an actual update?